src/infrastructure/transaction/SerializeTransactionToJSON.ts

```diff
@@ -54,7 +54,7 @@ export const SerializeTransactionToJSON = (transaction: Transaction): any => {
         case TransactionType.ADDRESS_ALIAS:
             return {
                 aliasAction: (transaction as AddressAliasTransaction).aliasAction,
-                namespaceId: (transaction as AddressAliasTransaction).namespaceId.toDTO(),
+                namespaceId: (transaction as AddressAliasTransaction).namespaceId.toHex(),
```

Comment:
Fair point here. The reason why we can / may not be provide original payload is due to the way transaction object get initiated. If a transaction is created vai DTO payload that should be fine. However, Transaction can also be created via buffer which is one case. There might be serials off chain manipulation agains an aggregate transaction for example, which does not touch the rest server at until final announcement. 

However, definitely worth raise a backlog issue for refactoring the legacy code. Perhaps to use the generated DTO objects rather than manual json objects.